packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapInitializer.java

```diff
@@ -94,14 +71,15 @@ public void onMapsSdkInitialized(MapsInitializer.Renderer renderer) {
     if (initializationResult != null) {
       switch (renderer) {
         case LATEST:
-          initializationResult.success("latest");
+          initializationResult.success(Messages.PlatformRendererType.LATEST);
           break;
         case LEGACY:
-          initializationResult.success("legacy");
+          initializationResult.success(Messages.PlatformRendererType.LEGACY);
           break;
         default:
           initializationResult.error(
-              "Unknown renderer type", "Initialized with unknown renderer type", null);
+              new Messages.FlutterError(
+                  "Unknown renderer type", "Initialized with unknown renderer type", null));
```

Comment:
Added `renderer.name()` as the details; we don't like to *change* errors since it could break people, but adding data that just wasn't there is safe for any but the most contrived case.